### DISCUSSION TITLE
How to monitor Oracle with Nightingale?

### DISCUSSION THREAD

**Discussion by superlaity** [ccfos/nightingale#1271] [General]
夜莺如何监控oracle？怎样才能连接上oracle数据库？怎么配置相关告警？
![image](https://user-images.githubusercontent.com/100276271/202076713-cdbafd6e-aa48-4096-9022-8ce3be1da217.png)

**Comment by FengZh61** [comment]
https://github.com/flashcatcloud/categraf/tree/main/inputs/oracle

需要安装oracle官方的client包

**Reply by superlaity** [reply]
官方的的client包已经弄好了，帮忙看看是这个吗
![image](https://user-images.githubusercontent.com/100276271/202098414-749f9557-a808-4b14-bf80-eaebaedea937.png)

**Reply by FengZh61** [reply]
> 官方的的client包已经弄好了，帮忙看看是这个吗 ![image](https://user-images.githubusercontent.com/100276271/202098414-749f9557-a808-4b14-bf80-eaebaedea937.png)

是这个
